libs/shared/utils/http/src/lib/models/pagination/paging-request.model.ts

```diff
@@ -34,18 +34,30 @@ export class PagingRequestModel implements SchemaModel<IPagingRequest, string> {
       this.fromSchema(paging);
     }
   }
-
+  private _navigate(queryParams: Params) {
+    this._router?.navigate([], {
+      queryParams: queryParams,
+      queryParamsHandling: 'merge',
+      relativeTo: this._activatedRoute,
+    });
+  }
+  private _buildQueryParams(): Params {
+    const queryParams: Params = {};
+    queryParams['pageNumber'] = this._pageNumber !== 0 ? this._pageNumber : null;
+    queryParams['pageSize'] = this._pageSize !== pageSizeOptions[0] ? this._pageSize : null;
+    queryParams['sortBy'] = this._sortBy ? this._sortBy : null;
+    queryParams['sortOrder'] = this._sortOrder !== 'ASC' ? this._sortOrder : null;
```

Comment:
You can use the or operator rather than the ternary if you want.

```suggestion
    queryParams['pageNumber'] = this._pageNumber || null;
    queryParams['pageSize'] = this._pageSize !== pageSizeOptions[0] ? this._pageSize : null;
    queryParams['sortBy'] = this._sortBy || null;
    queryParams['sortOrder'] = this._sortOrder || null;
```